docker/drupal/settings.overrides.php

```diff
@@ -16,6 +16,18 @@
   'collation' => 'utf8mb4_general_ci',
 ];
 
+/**
+ * Temporary migration source DB.
+ */
+$databases['migrate']['default'] = [
```

Comment:
Is this file for staging/prod?

Reply:
Yes. Those are the values that have been configured by SBIT on staging